examples/libming-CVE-2018-8962.sh

```diff
@@ -5,18 +5,23 @@ set -euo pipefail
 git clone https://github.com/libming/libming.git libming-CVE-2018-8962
 cd libming-CVE-2018-8962/; git checkout b72cc2f # version 0.4.8
 mkdir obj-aflgo; mkdir obj-aflgo/temp
-export SUBJECT=$PWD; export TMP_DIR=$PWD/obj-aflgo/temp
+export TMP_DIR=$PWD/obj-aflgo/temp
 export CC=$AFLGO/instrument/aflgo-clang; export CXX=$AFLGO/instrument/aflgo-clang++
 export LDFLAGS=-lpthread
 export ADDITIONAL="-targets=$TMP_DIR/BBtargets.txt -outdir=$TMP_DIR -flto -fuse-ld=gold -Wl,-plugin-opt=save-temps"
 echo $'decompile.c:398' > $TMP_DIR/BBtargets.txt
 ./autogen.sh;
-cd obj-aflgo; CFLAGS="$ADDITIONAL" CXXFLAGS="$ADDITIONAL" ../configure --disable-shared --prefix=`pwd`
+cd obj-aflgo; CFLAGS="-fcommon $ADDITIONAL" CXXFLAGS="$ADDITIONAL" ../configure --disable-shared --prefix=`pwd`
```

Comment:
What does -fcommon do?

Reply:
I have explained it in commit msg of 1e9c90c:
>  - *libming-CVE-2018-8962.sh* and *libming-CVE-2018-8962.sh*: Without "-fcommon" the linker called by clang will fail. See squaresLab/security-repair-benchmarks#19

Maybe the issue didn't exist when the scripts were created (4 years old at least).

And I have not yet figured out why this happened🙃